src/AndersonAcceleration.f90

```diff
@@ -72,183 +71,288 @@ MODULE AndersonAccelerationTypes
     PROCEDURE,PASS :: reset => reset_AndersonAccelerationType
 ENDTYPE AndersonAccelerationType
 
-!> Exception Handler for use in Anderson Acceleration
-TYPE(ExceptionHandlerType),SAVE :: eAndersonAccelerationType
-
 !> Name of module
-CHARACTER(LEN=*),PARAMETER :: modName='ANDERSONACCELERATIONTYPE'
-
+CHARACTER(LEN=*),PARAMETER :: modName='AndersonAccelerationTypes'
 !
 !===============================================================================
 CONTAINS
 !
 !-------------------------------------------------------------------------------
-!> @brief Initializes the Anderson Acceleration Type with a parameter list
+!> @brief Initializes the Anderson Acceleration Type with an input parameter list
 !> @param pList the parameter list
```

Comment:
looks like this is unneeded since you have `Params`

Reply:
Removed it